demo/project_demo13-DebeziumPostgres/run.py

```diff
@@ -11,72 +14,140 @@
+    # if they exist.
+
+    # Connector using JSON format
+    json_config = {
+        "connector.class": "io.debezium.connector.postgresql.PostgresConnector",
+        "database.hostname": "postgres",
+        "database.port": "5432",
+        "database.user": "postgres",
+        "database.password": "postgres",
+        "database.dbname": "postgres",
+        "table.include.list": f"{TEST_SCHEMA}.*",
+        "topic.prefix": "json",
+        "decimal.handling.mode": "string",
+        "time.precision.mode": "connect"
+    }
+
+    create_debezium_postgres_connector("inventory-connector-json", args.kafka_url_from_script, json_config, [f"json.{TEST_SCHEMA}.{TEST_TABLE}"])
+
+
+    # Connector using Avro format
+    avro_config = {
+        "connector.class": "io.debezium.connector.postgresql.PostgresConnector",
+        "database.hostname": "postgres",
+        "database.port": "5432",
+        "database.user": "postgres",
+        "database.password": "postgres",
+        "database.dbname": "postgres",
+        "table.include.list": f"{TEST_SCHEMA}.*",
+        "topic.prefix": f"avro",
+        "decimal.handling.mode": "string",
+        "time.precision.mode": "connect",
+        "key.converter": "io.confluent.connect.avro.AvroConverter",
+        "value.converter": "io.confluent.connect.avro.AvroConverter",
+        "key.converter.schemas.enable": "true",
+        "value.converter.schemas.enable": "true",
+        # Every connector needs a separate Postgres replication slot
+        "slot.name": "debezium_slot_1",
+        "key.converter.schema.registry.url": args.registry_url_from_connect,
+        "value.converter.schema.registry.url": args.registry_url_from_connect,
+        "time.precision.mode": "adaptive_time_microseconds"
+    }
+
+    create_debezium_postgres_connector("test-connector-avro", args.kafka_url_from_script, avro_config, [f"avro.{TEST_SCHEMA}.{TEST_TABLE}"])
+    run_feldera_pipeline(args.api_url, args.kafka_url_from_pipeline, args.registry_url_from_pipeline)
+
+def populate_database():
+    postgres_server = os.getenv("POSTGRES_SERVER", "localhost:6432")
+    with psycopg.connect(f"postgresql://postgres:postgres@{postgres_server}") as conn:
+        with conn.cursor() as cur:
+            # conn.autocommit = True
+            print(f"(Re-)creating test schema '{TEST_SCHEMA}'")
+            cur.execute(f"CREATE SCHEMA IF NOT EXISTS {TEST_SCHEMA}")
+            cur.execute(f"""
+                CREATE TABLE IF NOT EXISTS {TEST_SCHEMA}.{TEST_TABLE}(
+                        id INT PRIMARY KEY,
+                        bi BIGINT,
+                        s VARCHAR,
+                        d DOUBLE PRECISION,
+                        f REAL,
+                        i INT,
+                        b BOOLEAN,
+                        ts TIMESTAMP,
+                        dt DATE,
+                        json JSON,
+                        uuid UUID)""")
+            cur.execute(f"DELETE FROM {TEST_SCHEMA}.{TEST_TABLE}")
```

Comment:
this is the same as DELETE *?

Reply:
yes, but I don't think `delete *` is valid sql